docs/requirements.in

```diff
@@ -1 +1,2 @@
 Sphinx
+sphinx_ansible_theme
```

Comment:
as per earlier suggestion
```suggestion
sphinx_ansible_theme
..
```

Reply:
Commented above. We can try this approach after enabling dependabot, I want to be sure it works well with it too.